tests/python/test_perf_event.py

```diff
@@ -17,13 +17,13 @@ def test_cycles(self):
 BPF_HISTOGRAM(dist);
 int kprobe__sys_getuid(void *ctx) {
     u32 cpu = bpf_get_smp_processor_id();
-    u64 val = cnt1.perf_read(cpu);
+    u64 val = cnt1.perf_read(CUR_CPU_IDENTIFIER);
```

Comment:
that anti-opimization, since llvm will still emit call into bpf_get_smp_processor_id() in the line 19.